monkey/monkey_island/cc/ui/src/components/contexts/plugins/PluginsContext.tsx

```diff
@@ -85,15 +86,22 @@ export const PluginState = () :PluginsContextType => {
   const [installedPlugins, setInstalledPlugins] = useState([]);
   const [numberOfPluginsThatRequiresUpdate, setNumberOfPluginsThatRequiresUpdate] = useState(0);
 
+  useInterval(() => {
+    refreshAvailablePluginsAndNumberOfUpgradablePlugins(true);
+  }, 1000 * 60 * 60)
 
   useEffect(() => {
     refreshInstalledPlugins();
   }, []);
 
   useEffect(() => {
-    refreshAvailablePlugins().then(() => refreshNumberOfUpgradablePlugins());
+    refreshAvailablePluginsAndNumberOfUpgradablePlugins();
   }, [installedPlugins]);
 
+  const refreshAvailablePluginsAndNumberOfUpgradablePlugins = (forceRefresh = false) => {
```

Comment:
In general this is not a good name. If a name is "DoSomething1andDoSomething2" it means that there are 2 functions: "DoSomething1" and "DoSomething2". So we should just call "DoSomething1" and "DoSomething2" in a row instead of creating another function. But since there's a temporal coupling, I don't feel strongly about this

Reply:
you can change it if you have a better one